openblas/riscv64.patch

```diff
@@ -1,21 +1,26 @@
 --- PKGBUILD
 +++ PKGBUILD
-@@ -30,7 +30,7 @@ build() {
+@@ -30,9 +30,8 @@ build() {
      -DNO_AFFINITY=ON \
      -DUSE_OPENMP=1 \
      -DNO_WARMUP=1 \
 -    -DTARGET=CORE2 \
+-    -DNUM_THREADS=64 \
+-    -DDYNAMIC_ARCH=ON
```

Comment:
Why disable `DYNAMIC_ARCH`? Please leave a note here. Thanks.

Reply:
> Why disable DYNAMIC_ARCH? 

The option doesn't appear in official build guide of riscv64: https://github.com/OpenMathLib/OpenBLAS/blob/1b85b6a396c94e78c9ba14aafcdfd5c5da5a8bb2/README.md?plain=1#L208

It also cause a build failure like: https://archriscv.felixc.at/.status/log.htm?url=logs/vosk-api/vosk-api-0.3.50-5.log

Related issue: https://github.com/OpenMathLib/OpenBLAS/issues/949

> Please leave a note here. Thanks.

I'm sorry for foegeting that